includes/wikia/parser/templatetypes/TemplateTypesParser.class.php

```diff
@@ -35,6 +36,71 @@ public static function onFetchTemplateAndTitle( &$text, &$finalTitle ) {
 	}
 
 	/**
+	 * @desc change template wikitext according to template type
+	 *
+	 * @param $templateTitle
+	 * @param $templateWikitext
+	 * @return bool
+	 */
+	public static function onBraceSubstitution( $templateTitle, &$templateWikitext ) {
+		global $wgEnableTemplateTypesParsing, $wgArticleAsJson, $wgCityId;
+
+		wfProfileIn( __METHOD__ );
+
+		if ( $wgEnableTemplateTypesParsing
```

Comment:
This compound condition would probably go better extracted into a separate method